docker/simulation/carla_sample_node/carla_sample_node.Dockerfile

```diff
@@ -0,0 +1,63 @@
+ARG BASE_IMAGE=ghcr.io/watonomous/wato_monorepo/base:humble-ubuntu22.04
+
+################################ Source ################################
+FROM ${BASE_IMAGE} as source
+
+WORKDIR ${AMENT_WS}/src
+
+# Copy in source code 
+COPY src/simulation/carla_sample_node carla_sample_node
+COPY src/wato_msgs/sample_msgs sample_msgs
+COPY src/wato_msgs/simulation/common_msgs common_msgs
+COPY src/wato_msgs/simulation/embedded_msgs embedded_msgs
+COPY src/wato_msgs/simulation/path_planning_msgs path_planning_msgs
+
+# Carla specific messages
+RUN git clone --depth 1 https://github.com/ros-drivers/ackermann_msgs.git --branch ros2 && \
+    git clone --depth 1 https://github.com/ros-perception/image_common.git --branch $ROS_DISTRO && \
+    git clone --depth 1 https://github.com/carla-simulator/ros-carla-msgs.git --branch master
```

Comment:
git is working here?

Reply:
Yeah seems git is already installed on the base image